### ISSUE TITLE
todo: debias score classification thresholds

### ISSUE THREAD

**Issue by rkhaled98** [rkhaled98/Karmalutional-Network#5]

**Comment by rkhaled98** [comment]
"The output layer of the network will likely be a softmax classifier, which will likely use five categories. These will correspond to "terrible," "bad," "neutral," "good," and "top." The plan is to label the training data as such based on comment scores. It may be challenging to find appropriate comment score thresholds for demarcation without enough humans to classify comments into those categories. Only having one person determine the score could result in a network training to a single person's ideas of what a good comment is." - Nanak Khalsa

**Comment by rkhaled98** [comment]
Placeholder thresholds will be used in the meantime.
